.rust_alpha/ec2/README.md

```diff
@@ -0,0 +1,99 @@
+# AWS SDK for Rust code examples for Amazon EC2
+
+Amazon Elastic Compute Cloud (Amazon EC2) is a web service that provides resizable computing capacity—literally, servers in Amazon's data centers—that you use to build and host your software systems.
+
+## Purpose
+
+These examples demonstrate how to perform several Amazon EC2 operations using the alpha version of the AWS SDK for Rust.
+
+## Prerequisites
+
+You must have an AWS account, and have configured your default credentials and AWS Region as described in [https://github.com/awslabs/aws-sdk-rust](https://github.com/awslabs/aws-sdk-rust).
+
+## Running the code
+
+### describe-instances
+
+This example lists the state of one or all of your Amazon EC2 instances
+
+`cargo run --bin describe-instances -- [-i INSTANCE-ID] [-d DEFAULT-REGION] [-v]`
+
+- _INSTANCE-ID_ is the ID of an instance to describe.
+  If this argument is not supplied, the state of all instances is shown.
+- _DEFAULT-REGION_ is optional name of a region, such as __us-east-1__.
+  If this value is not supplied, the region defaults to __us-west-2__.
+- __-v__ display additional information.  
+
+### ec2-helloworld
+
+This example lists the state of one or all of your Amazon EC2 instances
+
```

Comment:
What's the difference between describe-instances and ec2-helloworld? They seem to be doing the same thing.

Reply:
Oops, copy and paste error. I've fixed it.